### ISSUE TITLE
New, extra padding in output area result

### ISSUE THREAD

**Issue by parente** [jupyter-incubator/dashboards_server#227]
The cougars used to top-align. Now they don't.

![screen shot 2016-05-12 at 9 18 08 am](https://cloud.githubusercontent.com/assets/153745/15215842/87a8137c-1822-11e6-933c-2dc20a7c39bd.png)

Reason:

![screen shot 2016-05-12 at 9 18 13 am](https://cloud.githubusercontent.com/assets/153745/15215850/90355e32-1822-11e6-9c2f-e9c70a12b227.png)

Do we care?

**Comment by dalogsdon** [comment]
Hmm Python vs Markdown images.Visually it would be nice but it could get really tedious fixing cases like  this, especially if the new Jupyter/Phosphor JS libs keep changing. I would say this could be something to look at in the future once things are more stable.

**Comment by parente** [comment]
Yes. I'm not putting this into a milestone. Just wanted to note it in case one day we can do something about it easily.
